default/test_package/conanfile.py

```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
 
 from conans import ConanFile, CMake
```

Comment:
Could you please add `tools`? Since we check for cross-building, there is no tools imported (bug)

Reply:
Ok, will do